### DISCUSSION TITLE
Contact Question

### DISCUSSION THREAD

**Discussion by ats5662** [idaholab/moose#20340] [Q&A General] [answered]
Is there an option in moose to define general contact or self contact? I am looking to run a simulation similar to the picture below. Thanks!

![image](https://user-images.githubusercontent.com/84894791/154611878-55a28198-a584-45eb-87b4-9aacab68174e.png)

**Comment by jiangwen84** [comment, marked as answer]
Unfortunately, MOOSE does not support self contact at this moment.
